src/System.Runtime.Intrinsics.X86/src/Configurations.props

```diff
@@ -0,0 +1,9 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Windows_NT;
```

Comment:
Did we need both Windows_NT and Unix configurations for some reason? I think this should work with just a "netcoreapp" configuration. Other than that it looks good to me to merge.

Reply:
This change is from @weshaggard 's suggestion https://github.com/dotnet/corefx/pull/23489#discussion_r137863998 , for  "just a 'netcoreapp' configuration" causes build errors.

Reply:
Thanks for the reminder. That makes sense.